reposilite-site/docs/reverse-proxy-ssl.md

```diff
@@ -0,0 +1,86 @@
+---
+id: reverse-proxy-ssl
+title: Reverse proxy ssl
+sidebar_label: Reverse proxy ssl
+---
+
+Lots of people like to use a reverse proxy like nginx with reposilite. This is a page on
+how to setup Nginx with SSL.
+
+Source: <https://gist.github.com/Saladoc/8c17bda7b131c4fade9b2189ea5050d8>
+
+## Step 1
+First, install and setup Reposilite. Make sure to setup Reposilite to listen on port 8080 (or anything other than 80 and 443).
+
+Then, install nginx, openssl, and certbot [using snapd](https://snapcraft.io/docs/installing-snapd/) `sudo snap install certbot --classic` and `sudo ln -s /snap/bin/certbot /usr/bin/certbot`
+
+## Step 2
+Next you have to generate your certificates. To do this you will need a valid domain name and have your server pointed at it. 
+Run `sudo certbot certonly --standalone` and follow the instructions.
```

Comment:
If you're doing it this way, the paths to the certificates need to be adjusted manually (and the "Managed by Certbot" comments removed). If you install the nginx plugin for certbot as well, it can do the configuration for you.

Reply:
The nginx plugin didn’t work for me, however, using certonly will also schedule a cronjob.